package.json

```diff
@@ -79,9 +79,11 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-standard": "^5.0.0",
     "eslint-plugin-vue": "^7.9.0",
+    "express-ws": "^4.0.0",
     "istanbul-instrumenter-loader": "^3.0.1",
     "istanbul-lib-coverage": "^3.0.0",
     "json-server": "^0.16.3",
+    "nodemon": "^2.0.7",
```

Comment:
The alternative to `nodemon` is to write something like

```js
const fs = require('fs')

const someFile = __filename
fs.watch(someFile, () => { ... })
```

Which gets trickier when you want to monitor multiple files (e.g. `index.json` for `json-server`, plus any of the `JSON` files, or even Cylc UI JS code...)